### ISSUE TITLE
Is this project still maintained?

### ISSUE THREAD

**Issue by oliviercailloux** [beijunyi/ParallelGit#11]
I have been implementing a read-only access to git as a Java-7 nio FS, only to discover recently that your project already does it, and seems more complete as it also implements write access.

It seems like the source code is of good quality, and with lots of unit tests, so I’d be most interested in using it in my project. It only lacks some docs, as apparently there is no Javadoc at all in the source code (but the code seems clear enough to be understandable).

This repository has got 110 stars and 22 forks, which does suggest that the project raises some interest. It is [used](https://mvnrepository.com/artifact/com.beijunyi/parallelgit-filesystem/2.0.0/usages) by two artifacts published on Maven Central: [The Modern Way Server Core](https://github.com/themodernway/themodernway-server-core), which seems abandoned (last updated June 2018), and [browserbox-maven-plugin](https://bitbucket.org/prunge/browserbox-maven-plugin/), updated a few days ago and still [using](https://bitbucket.org/prunge/browserbox-maven-plugin/raw/766d33d3057d63629efc90efcce4d0bf8c39479b/browserbox-maven-plugin/pom.xml) ParallelGit, AFAICS.

However, this project has not been updated since 3 years+. Is it abandoned? Is there some hope to see a release 2.1? I see that some commits have been added since 2.0.0. Also, some of the forks visible on GitHub are one commit ahead of the basis, which suggests some patches are available and could be integrated. (Please comment here, anyone who has forked this project and would be willing to contribute back.)

In summary, any information about the current status of this project would be most welcome!
